napari/layers/labels/labels.py

```diff
@@ -939,12 +933,7 @@ def _update_thumbnail(self):
         like adjusting gamma or changing the data based on the contrast
         limits.
         """
-        if not self.loaded:
-            # ASYNC_TODO: Do not compute the thumbnail until we are loaded.
-            # Is there a nicer way to prevent this from getting called?
-            return
```

Comment:
Is this removal related to this PR or just a forgotten leftover from previous changes?